skimage/_shared/fast_exp.h

```diff
@@ -3,8 +3,10 @@
  * Reference [2]: https://doi.org/10.1162/089976600300015033
  * Additional improvements by Leonid Bloch. */
 
+#include <stdint.h>
+
 /* use just EXP_A = 1512775 for integer version, to avoid FP calculations */
-#define EXP_A (1512775.3951951856938)  /* 2^20*ln2 */
+#define EXP_A (1512775.3951951856938)  /* 2^20/ln2 */
```

Comment:
Since this fix is unrelated, please put it to a separate commit. It will make things easier if the functional part of this commit will ever need to be reverted, and in general making the commits as specific as possible is a good practice.

Reply:
We squash many PRs. @rfezzani if you want us to merge each commit, you will have to rebase and clean the history a little as suggested by blochl

Reply:
I don't really care if the PR is squashed. I applied blochl suggestion to not be lazy :-)

Reply:
Thanks!!
Not sure if squashing the entire PR is a good idea for later references/fixes/feature history, but hey - not my call. :)